TypeScript errors thrown from `undici` when building project

Trying to build a TS project which references the latest v4 version (4.0.0-alpha.3) will result in TS errors being thrown:

```shell
node_modules/undici/types/client.d.ts:5:8 - error TS1259: Module '"C:/Users/hossamnasr/ms/azure/durable/js-sdk/node_modules/undici/types/connector"' can only be default-imported using the 'esModuleInterop' flag

5 import buildConnector, {connector} from "./connector";
         ~~~~~~~~~~~~~~

  node_modules/undici/types/connector.d.ts:4:1
    4 export = buildConnector
      ~~~~~~~~~~~~~~~~~~~~~~~
    This module is declared with using 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.

node_modules/undici/types/dispatcher.d.ts:6:13 - error TS1259: Module '"C:/Users/hossamnasr/ms/azure/durable/js-sdk/node_modules/undici/types/readable"' can only be default-imported using the 'esModuleInterop' flag

6 import type BodyReadable from './readable'
              ~~~~~~~~~~~~

  node_modules/undici/types/readable.d.ts:4:1
    4 export = BodyReadable
      ~~~~~~~~~~~~~~~~~~~~~
    This module is declared with using 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.

node_modules/undici/types/handlers.d.ts:1:8 - error TS1259: Module '"C:/Users/hossamnasr/ms/azure/durable/js-sdk/node_modules/undici/types/dispatcher"' can only be default-imported using the 'esModuleInterop' flag

1 import Dispatcher from "./dispatcher";
         ~~~~~~~~~~

  node_modules/undici/types/dispatcher.d.ts:12:1
    12 export = Dispatcher;
       ~~~~~~~~~~~~~~~~~~~~
    This module is declared with using 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.


Found 3 errors.
```
#### Repro steps

Provide the steps required to reproduce the problem:

1. Create a new TypeScript app with the following `tsconfig.json`:
```Json
{
    "compilerOptions": {
        "module": "commonjs",
        "target": "es6",
        "outDir": "dist",
        "rootDir": ".",
        "sourceMap": true,
        "strict": false,
        "baseUrl": "./"
    }
}
```
3. Install `@azure/functions` version `4.0.0-alpha.3`
4. Run `npm run build`
5. You get the above TypeScript errors

#### Expected behavior

Builds without errors

#### Actual behavior

See the above TypeScript errors

#### Known workarounds

Pin `undici` to `5.9.1`

--- a/package.json
+++ b/package.json
@@ -42,7 +42,7 @@
     },
     "dependencies": {
         "long": "^4.0.0",
-        "undici": "~5.10.0"
+        "undici": "^5.13.0"
     },
     "devDependencies": {
         "@types/chai": "^4.2.22",

--- a/test/parsers/parseForm.test.ts
+++ b/test/parsers/parseForm.test.ts
@@ -152,7 +152,7 @@ Content-Type: text/html; charset=UTF-8
             expect(parsedForm.has('htmlfile')).to.equal(true);
             const file = <File>parsedForm.get('htmlfile');
             expect(file.name).to.equal('test.html');
-            expect(file.type).to.equal('text/html; charset=utf-8');
+            expect(file.type).to.equal('text/html;charset=utf-8');
             expect(await file.text()).to.equal('<h1>Hi</h1>');
         });
 

